modules/system/models/MailSetting.php

```diff
@@ -55,16 +55,17 @@ class MailSetting extends Model
     public function initSettingsData()
     {
         $config = App::make('config');
-        $this->send_mode = $config->get('mail.driver', static::MODE_MAIL);
+        $mailers = $config->get('mail.mailers', []);
```

Comment:
```suggestion
        $mailers = $config->get('mail.mailers', [
            'sendmail' => ['path' => $config->get('mail.sendmail', '/usr/sbin/sendmail')],
            'smtp' => [
                'host' => $config->get('mail.host'),
                'port' => $config->get('mail.port', 587),
                'username' => $config->get('mail.username'),
                'password' => $config->get('mail.password'),
            ],
        ]);
```

Reply:
Oh, ok, you're allowing the old config to be used, that's interesting.